signature-pad/src/main/java/com/github/gcacace/signaturepad/utils/Svg.java

```diff
@@ -0,0 +1,28 @@
+package com.github.gcacace.signaturepad.utils;
+
+public class Svg {
```

Comment:
What do you think about renaming this class in something like `SvgCurve`, then create an instance of it and store inside it the list of `<path/>`, finally create a `build()` method to apply header, footer and return the SVG XML.